paint/PAINT_Eraser.h

```diff
@@ -3,7 +3,7 @@
 
 namespace paint
 {
-	class Eraser final :
+	class Eraser :
```

Comment:
Why remove `final`?

Reply:
Resharper complained, I've added it back